File: Python/Cogs/Fun/fun.py
import random
from random import random

import discord
from discord.ext import commands
from discord.commands import Option  #Importing the packages
from discord.commands import slash_command


class Fun(commands.Cog):
    def __init__(self, bot):#to Initialise
        self.bot = bot

    @slash_command(guild_ids=[...])
    async def eightball(self, ctx, question: Option(str)):
        ballresponse = [
            "Yes", "No", "Take a wild guess...", "Very doubtful",
            "Sure", "Without a doubt", "Most likely", "Might be possible",
            "You'll be the judge", "no... (╯°□°）╯︵ ┻━┻", "no... baka",
            "senpai, pls no ;-;","i think....","gg","I-\ndont know what to say"
        ]

        answer = random.choice(ballresponse)
        await ctx.respond(f"🎱 **Question:** {question}\n**Answer:** {answer}")
    
    @slash_command(guild_ids=[...])
    async def hotcalc(self, ctx, user: Option(discord.Member)):    
        random.seed(user.id)
        r = random.randint(1, 100)
        hot = r / 1.17

        if hot > 75:
            emoji = "💞"
        elif hot > 50:
            emoji = "💖"
        elif hot > 25:
            emoji = "❤"
        else:
            emoji = "💔"    

        await ctx.respond(f"**{user.name}** is **{hot:.2f}%** hot {emoji}")            

def setup(bot):
    bot.add_cog(Fun(bot))        

File: Javascript/commands/Fun/fun.js
const Discord = require("discord.js")
const mathutils = require("../mathutils")

function useCommands(msg, prefix) {
    if (msg.content.toLowerCase().startsWith(prefix + "eightball")) { // Check if the message starts with the prefix and eightball so users can use the command
        const ballresponce = [
            "Yes", "No", "Take a wild guess...", "Very doubtful",
            "Sure", "Without a doubt", "Most likely", "Might be possible",
            "You'll be the judge", "No... (╯°□°）╯︵ ┻━┻", "No!",
            "bruh no","Maybe, 👀","gg","I don't know"
        ] // Tells our program what answers we want from the bot

        const answer = ballresponce[mathutils.getRandomInt(0, ballresponce.length - 1)] // Gets a random answer from the responces section

        msg.channel.send(answer) // Sends the answer (when I ask if I am a good person it always says no (Yaumama writing) )
    }

    if (msg.content.toLowerCase().startsWith(prefix + "hotlevel")) { // Check if the message is the prefix and hotlevel
        const hot = mathutils.getRandomInt(0, 100) // Get the percentage of hot the person is
        let emoji // Make an unsigned variable for us to set later

        if (hot > 75) { // Check if the hot level is above these levels and add emojis accordingly
            emoji = "💞"
        } else if (hot > 50) {
            emoji = "💖"
        } else if (hot > 25) {
            emoji = "❤"
        } else {
            emoji = "💔"
        }

        msg.channel.send("**" + msg.member.displayName + "** is " + hot + "% hot! " + emoji) // Send the hot level and emoji
    }
}

module.exports = {useCommands}
